2.6.0/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+namenode:
+    image: gelog/hadoop
+    ports: 
+        - "50070:50070"
+    command: hdfs namenode
+    hostname: hdfs-namenode
+
+datanode:
+    image: gelog/hadoop
+    command: hdfs datanode
+    ports:
+        - "50075"
```

Comment:
I don't know if it is possible to do that, but can we add an inline comments here explaining something along the lines of "The host port is randomly assigned by Docker, to allow multiple DataNodes on the same host".